src/main/resources/default/extensions/tycho-page-menu/footer.html.pasta

```diff
@@ -0,0 +1,30 @@
+<i:arg type="String" name="point"/>
+<i:pragma name="priority" value="1000"/>
+
+<i:switch test="@point">
+    <i:block name="footer-right-start">
+        <t:permission permission="flag-logged-in">
+            <span class="d-none d-xl-inline-block text-muted small mr-2 pr-2 br-gray">
+               <b>@user().getUserName()</b>
+               <i:if test="isFilled(user().getTenantName())">(@user().getTenantName())</i:if>
+            </span>
+        </t:permission>
+    </i:block>
+    <i:block name="footer-right-end">
+        <t:permission permission="flag-logged-in">
+            <i:if test="user().is(sirius.biz.tenants.Tenant.class)">
+                <span class="d-none d-md-inline-block text-muted small ml-2 pl-4 pr-0 bl-gray">
+                    <a id="supportLink" href="javascript:copySupportLink()" class="text-sirius-gray-dark"><i class="fa fa-link"></i></a>
```

Comment:
can we bind this action in the script portion instead of having javascript call directly in the href attribute?

Reply:
maybe even also add the url itself to the href attribute, this additionaly enables right click --> copy link url which might prevent some misunderstandings for the support staff if both work :)

Reply:
what's the benefit?

Reply:
In short it presents the user with less options, and increases the chances that people will successfully trigger the function.

The presence of this element tells the browser that this is a link with a destination. With that, the browser will show the Open In New Tab / Window function (also triggered when you use shift+click or middle mouse) and the option to copy the link. Doing so will result in opening the same page without the function being called so the actual link is not being copied(resulting in user frustration). Also this shows the ‘ javascript:copySupportLink()’ on hover in the browser status bar, which is confusing and not user friendly imo. Maybe we shouldn’t even use <a> tag but a span/button instead?

Reply:
Also, this goes against the principle of seperating structure(html) from code(script)